src/main/java/com/manardenza/utils/Injector.java

```diff
@@ -14,14 +17,19 @@
 public class Injector {
     private static File hotelsDatabaseFile = new File("src/main/resources/databaseFiles/databaseHotelsFile.bin");
     private static File usersDatabaseFile = new File("src/main/resources/databaseFiles/databaseUsersFile.bin");
+    private static File reservationDatabaseFile = new File("src/main/resources/databaseFiles/databaseReservationsFile.bin");
     @Getter
     private static HotelDaoImpl hotelDao = new HotelDaoImpl(hotelsDatabaseFile);
     @Getter
     private static UserDaoImpl userDao = new UserDaoImpl(usersDatabaseFile);
+    @Getter
+    private static ReservationDaoImpl reservationDao = new ReservationDaoImpl(reservationDatabaseFile);
     private static HotelService hotelService = new HotelService(hotelDao);
     private static UserService userService = new UserService(userDao, CurrentUser.getInstance());
+    private static ReservationService reservationService = new ReservationService(reservationDao);
     private static HotelController hotelController = new HotelController(hotelService);
     private static UserController userController = new UserController(userService);
+    private static ReservationController reservationController = new ReservationController(reservationService);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "reservationController" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)